mitmproxy/certs.py

```diff
@@ -281,12 +300,18 @@ def _fix_legacy_sans(sans: Iterable[x509.GeneralName] | list[str]) -> x509.Gener
         return x509.GeneralNames(sans)
 
 
+@dataclass
+class RevocationInfo:
+    crl_distribution_points: list[str] = field(default_factory=list)
+
+
 def dummy_cert(
     privkey: rsa.RSAPrivateKey,
     cacert: x509.Certificate,
     commonname: str | None,
     sans: Iterable[x509.GeneralName],
     organization: str | None = None,
+    revocation: RevocationInfo = RevocationInfo(),
```

Comment:
Can we simplify this to `crl_url: str | None = None` please?

Reply:
So the dummy cert will only be able to account for one of the CRLs in the upstream cert? I guess that should be fine??

Reply:
Oh alright, will revise it to just `str`, sounds good